cmd/request/request_test.go

```diff
@@ -0,0 +1,70 @@
+package request_test
+
+import (
+	"testing"
+
+	"github.com/pingidentity/pingctl/internal/testing/testutils"
+	"github.com/pingidentity/pingctl/internal/testing/testutils_cobra"
+)
+
+// Test Request Command Executes without issue
+func TestRequestCmd_Execute(t *testing.T) {
+	err := testutils_cobra.ExecutePingctl(t, "request",
```

Comment:
Go's Example testing requires an exact match, and cannot be tested via regex. Since Pingone outputs transaction IDs on every API call, checking stdout is not possible via that method.

Is that what you were referring to?